docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/basics/send-assets-to-l1.mdx

```diff
@@ -16,6 +16,12 @@ import AboutAccounts from '../../../_admonitions/_about-accounts.md';
 
 <AboutAccounts/>
 
+:::info
+
+Be aware that SMR has 6 decimals the L2 representation 18. So if you are don't send enough fractions of a Shimmer to L1 the transation will fail. 
```

Comment:
Please don't try to write sentences at midnight, it seems to affect your understanding of how languages work :trollface:

Reply:
Yup, it's not very clear, how about:
```suggestion
ShimmerEVM has 18 decimal places, while Shimmer has 6. This means that any decimals beyond the 6th will be ignored by Shimmer, even though you can see them on ShimmerEVM. Please keep this in mind while sending tokens. 
```

Reply:
```suggestion
ShimmerEVM has 18 decimal places, while Shimmer has 6. This means that any decimals beyond the 6th will be ignored by Shimmer, even though you can see them on ShimmerEVM. Please keep this in mind while sending tokens, as your transaction will fail if its value is less than 1 glow or what is needed to cover its [storage deposit](/learn/protocols/stardust/core-concepts/storage-deposit/).
```

Reply:
```suggestion
ShimmerEVM has 18 decimal places, while Shimmer has 6. This means that any decimals beyond the 6th will be ignored by Shimmer, even though you can see them on ShimmerEVM. Please keep this in mind while sending your tokens to L1.
```